src/test/java/hudson/remoting/PrefetchTest.java

```diff
@@ -36,7 +35,6 @@
 /**
  * @author Kohsuke Kawaguchi
  */
-@Ignore
```

Comment:
Disabled in #387, yet consistently passing for me, and this annotation does not even work since #593, so trying to run it again. Whatever it is that it is testing; from https://github.com/jenkinsci/remoting/blob/b5eba8204c1e200a6a225e806e80c86e36f45710/src/test/java/hudson/remoting/PrefetchTest.java#L46-L47 that may be nothing and maybe it should simply be deleted.